mintpy/ifgram_inversion.py

```diff
@@ -196,6 +198,21 @@ def cmd_line_parse(iargs=None):
 
     inps.timeseriesFile, inps.tempCohFile = inps.outfile
 
+    # --config option
+    inps.config = inps.config.lower()    # use lowercase all the time
+    if inps.config == 'no':
```

Comment:
Wouldn't it be better to move this condition to after the config auto values have been sourced so we can just say `if inps.config` rather than having to do this string comparison?

Reply:
The auto value from smallbaselineApp.cfg file is replaced within read_template2inps(), so yes, this is after that. I changed the default value for --config option in this script to "no" so that both default value from ifgram_inversion.py and smallbaselineApp.cfg file are the same. Then, we translate the config=='no' here to a more meaningful value.

Reply:
The string comparison feels unclean to me. I don't see it anywhere else in the codebase, even for parameters that use 'no' as default values.

Reply:
As long as it's clear to users and works, it's fine with me. We can commit this and come back to it later when we have a cleaner solution.

Reply:
`inps.config = inps.config.lower()` is to ensure it works for scenarios when one entered upper case config. Just a robust checking.

It's a little bit of confusion in smallbaselineApp.cfg file that we use upper case for cluster and lower case for config for the default values. Thus, I adjust the comments in file to all UPPER case except for "no", to be more consistent.

Reply:
I was actually going to fix this so that the cluster type was capitalization-agnostic.

Reply:
That would be even better!